src/rulesets/rest/2022-05-25/json-api-rules/__tests__/resource-object-rules.test.ts

```diff
@@ -121,6 +121,111 @@ describe("resource object rules", () => {
         expect(results.every((result) => result.passed)).toBe(true);
       },
     );
+
+    test.each(["uuid", "uri"])(
+      "passes when PATCH request body for a relationship is of the correct form",
```

Comment:
nit: I cannot tell what the "correct form" is unless I read the body of the test, so I don't know if the test is accurate. I'll only comment this once to reduce noise

Reply:
What would be your recommended improvement here? Pull out the object from within the test and place it in a variable at the start of the test block?